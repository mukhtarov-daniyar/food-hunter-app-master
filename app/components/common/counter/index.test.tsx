import React from 'react';
import renderer from 'react-test-renderer';
import Counter from './index';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Counter decrement={jest.fn()} increment={jest.fn()} quantity={1} />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
