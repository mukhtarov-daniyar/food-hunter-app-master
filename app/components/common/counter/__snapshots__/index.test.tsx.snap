// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "space-between",
      },
      null,
    ]
  }
>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "opacity": 1,
        "padding": 19.23076923076923,
      }
    }
  >
    <Text
      style={
        {
          "fontSize": 35.416666666666664,
        }
      }
    >
      -
    </Text>
  </View>
  <Text
    style={
      {
        "fontSize": 35.416666666666664,
        "paddingHorizontal": 19.23076923076923,
      }
    }
  >
    1
  </Text>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "opacity": 1,
        "padding": 19.23076923076923,
      }
    }
  >
    <Text
      style={
        {
          "fontSize": 35.416666666666664,
        }
      }
    >
      +
    </Text>
  </View>
</View>
`;
